### ISSUE TITLE
Excessively large, clipped fonts, and oversized toolbar icon

### ISSUE THREAD

**Issue by hackel** [paralect/robomongo#1084]
Oddly, this just started happening, but I haven't changed my font sizes.  I think this is related to #443 and #649, but this seemed like a slightly different twist with the toolbar icon and all dialogues being oversized  as well.

![image](https://cloud.githubusercontent.com/assets/583677/14901017/ef36d8d6-0d58-11e6-98d1-f40ef9015004.png)

I'm running Ubuntu 16.04 on a 128 DPI display (1920x1080).
System fonts: Default: Cantarell Regular 7.4, Mono: Ubuntu Mono 9.
(In case it makes any difference, to achieve the proper 128 dpi, I had to use a text-scaling-factor of 1.16 with a UI scale of .875.)

**Comment by anorsich** [comment]
@hackel Thanks for reporting that issue. It looks like you are first one who tested on the Ubuntu 16.04, thank you for that! @juliashibalko Can we verify if we can replicate that issue and assign appropriate labels, good chance to taste Ubuntu 16.04 :)

**Comment by juliashibalko** [comment]
@anorsich Sure! I'm on the way :)

**Comment by juliashibalko** [comment]
@hackel thank you for detailed information on your system! I set all values as you've described above but fonts look good for me now. I'll continue keeping an eye on this issue and will let you know about any updates.

**Comment by hackel** [comment]
I also wasn't able to reproduce this in a guest session, so I'm not sure what's going on.  I figured out how to manually set the font family/size in robomongo.json and that fixed the issue for me, at least.

**Comment by greenlaw110** [comment]
I am on Mint 17.3, and for long time I got this kind of Look and Feel:
![image](https://cloud.githubusercontent.com/assets/216930/15136511/89013d06-16c5-11e6-8399-9d38c812094d.png)

My operating system font setting:

![image](https://cloud.githubusercontent.com/assets/216930/15136529/a485f0e4-16c5-11e6-8434-ea3d4dbce3b2.png)

**Comment by hackel** [comment]
Okay, after a little more research I was finally able to reproduce this!  I have a dual-display setup, with a built-in 17" LCD and 20" external display.  I forgot that I had also added startup script to correctly set the screen dimensions and DPI: `xrandr --fbmm 382x215`.  This is only for the internal display, however, which was causing the DPI to get messed up with the external display attached (DPI reported as 234x128 instead of 128x128).  Once I fixed the dimensions to account for the second display, the DPI values were again equal and Robomongo's fonts started behaving again.

It's curious that only Robomongo was affected and not the rest of my system.  I assume this has something to do with the way Qt renders fonts vs. GTK.
